curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo/65004ba581d03d1d5628b41c.md

```diff
@@ -0,0 +1,770 @@
+---
+id: 65004ba581d03d1d5628b41c
+title: Step 56
+challengeType: 0
+dashedName: step-56
+---
+
+# --description--
+
+You've retrieved the task item(s) now but they're still not reflecting in the UI when the page loads, but then they appear when you add a new task.
+
+To fix this issue, you can check if there's a task inside `taskData` and then call the `addTaskToTaskContainer()` or call the `addTaskToTaskContainer()` function inside an immediately invoked function expression (IIFE).
```

Comment:
This might be the first time an IIFE is introduced.

If so, then we should define it and and show a code example

Reply:
It's better not to talk about IIFE at all then, since the instruction didn't say it should be used.